packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -1077,7 +1077,7 @@ export class DataViewExpander {
                     description = value.description;
                 }
 
-                // Peer Relationship Attribute
+                // Peer shared RelationshipAttribute
```

Comment:
instead of changing these comments, please yeet them

Reply:
Maybe in case of IdentityAttributeDVOs they actually are a good translation (PeerAttribute = Peer shared IdentityAttribute, SharedToPeerAttribute = Own shared IdentityAttribute). Of course renaming them would even be better, but breaking.